jekyll.gemspec

```diff
@@ -36,4 +36,5 @@ Gem::Specification.new do |s|
   s.add_runtime_dependency('rouge', '~> 1.7')
   s.add_runtime_dependency('jekyll-sass-converter', '~> 1.0')
   s.add_runtime_dependency('jekyll-watch', '~> 1.1')
+  s.add_runtime_dependency("pathutil", "~> 0.9")
```

Comment:
>  scroll down a few lines

@envygeeks - ha...my :eyes: missed the indentation level. Thanks for the pointer. In general I like the idea of a centralized and well tested library for this kind of stuff. My only real reservations are whether there are edge cases between how `PathUtil` will handle things vs. how `Pathname` would handle things. We have run into similar concerns (and issues) with differences between URL parsers (ex. how does `URI.parse` handle a URL vs. `Addressable::URI.parse`). Since this  "responds to all methods that Pathname provides" would it make sense to have a test suite that tracks the tests in https://github.com/ruby/ruby/blob/trunk/test/pathname/test_pathname.rb to ensure compatibility (with explicit comments on when tests deviate and why)?

Related to the `Pathname` compat...the first thing I tried didn't work:

``` ruby
require 'pathname'
Pathname.new("/etc/").cleanpath
=> #<Pathname:/etc>
...
require 'pathutil'
Pathutil.new("/etc/").cleanpath
NoMethodError: undefined method `cleanpath' for #<Pathutil:/etc/>
Did you mean?  realpath
```